Assets/HealthBar.cs

```diff
@@ -0,0 +1,21 @@
+using System.Collections;
```

Comment:
- [ ] Move this script to the `Assets/Scripts` folder. (Maybe to `Assets/Scripts/UI`.)